src/services/GeolocationService.js

```diff
@@ -0,0 +1,20 @@
+const GeolocationService = {
+  getCurrentPosition: function () {
+    return new Promise((resolve, reject) => {
+      const position = navigator.geolocation.getCurrentPosition(
+        position => this.getPositionSuccess(position, resolve),
+        () => this.getPositionError(reject)
+      );
+    });
+  },
+
+  getPositionSuccess: (position, resolve) => {
```

Comment:
For now I don't see any added value with these getPositionSuccess and getPositionError